webpack/scenes/AlternateContentSources/ACSConstants.js

```diff
@@ -18,4 +18,27 @@ export const ACS_TYPE_TRANSLATIONS_ENUM = {
   [FILE]: 'file',
 };
 
+export const isValidUrl = (urlString) => {
```

Comment:
okay, technically it's a `const` but I feel like these fit better in a `_helpers.js` file rather than a `Constants.js` file :smile:

Reply:
Ya..I was thinking of a helpers file..Let me put these in there..